src/linux.js

```diff
@@ -1,16 +1,31 @@
 const exec = require('@actions/exec');
 
 const WORKING_DIR = '/tmp/diff-pdf-src';
-const REQUIRED_PACKAGES = [
+const REQUIRED_PACKAGES_BASE = [
   'make',
   'automake',
   'g++',
   'libpoppler-glib-dev',
   'poppler-utils',
-  'libwxgtk3.0-gtk3-dev'
 ];
 
 export async function setup(version) {
+  // Get Ubuntu version codename (e.g., jammy, noble)
+  let ubuntuVersion = '';
+  await exec.exec('lsb_release -cs', [], {
+    listeners: {
+      stdout: (data) => {
+        ubuntuVersion += data.toString().trim();
+      }
+    }
+  });
+
+  const wxgtkPackage = (ubuntuVersion === 'jammy')
+    ? 'libwxgtk3.2-dev'
+    : 'libwxgtk3.0-gtk3-dev';
```

Comment:
It seems the condition is reversed. I believe the correct code should be: 
```suggestion
  const wxgtkPackage = (ubuntuVersion === 'jammy')
    ? 'libwxgtk3.0-gtk3-dev'
    : 'libwxgtk3.2-dev';
```
Would you mind confirming?

Reply:
Hi @hidakatsuya ,

Thank you for pointing out the reversed condition and sorry for the delayed response. 
I’ve just updated the code according to your suggestion.